opsdroid/web.py

```diff
@@ -29,10 +79,34 @@ def __init__(self, opsdroid):
         self.web_app = web.Application()
         self.runner = web.AppRunner(self.web_app)
         self.site = None
+        self.command_center = self.config.get("command-center", {})
         if not self.config.get("disable_web_index_handler_in_root", False):
             self.web_app.router.add_get("/", self.web_index_handler)
             self.web_app.router.add_get("", self.web_index_handler)
 
+        self.excluded_keys = EXCLUDED_CONFIG_KEYS + self.command_center.get(
+            "excluded-keys", []
+        )
+        self.expected_header = self.command_center.get("token")
+        if self.command_center:
+            self.web_app.router.add_get("/connectors", self.connectors_handler)
+            self.web_app.router.add_get("/connectors/", self.connectors_handler)
```

Comment:
I wouldn't bother with the routes with the training slash. This only matters for the root route as browsers may append a slash.